zzz_modular_syzygy/code/modules/scrap/oldificator.dm

```diff
@@ -0,0 +1,6 @@
+/obj/item/weapon/computer_hardware/hard_drive/portable/design/make_old()
+	.=..()
+	if (.)
+		if(prob(90))
```

Comment:
Indent here looks messed up. Also, combine this with the if check under it. `if(license > 0 && prob(90))`